src/library_webgpu.js

```diff
@@ -391,6 +404,7 @@ var LibraryWebGPU = {
       'compute-shader-invocations',
     ],
     PowerPreference: [
+      'undefined',
```

Comment:
I think this should be unquoted `undefined` rather than `'undefined'`. (Maybe this was a dawn generator error?)